README.md

```diff
@@ -323,10 +366,7 @@ page.uploadFile('#selector', '/path/to/file').then(function() {
 });
 ```
 
-## Advanced
-Methods below are for advanced users. Most people won't need these methods.
```

Comment:
Why remove this? It truly is for advanced users. `defineMethod` should never be used because most methods are defined statically instead in `page.js`? `page.addCookie` is preferred while `page.invokeAsyncMethod('addCookie')` is not.